collect.py

```diff
@@ -203,6 +203,19 @@ def screenshot(driver, file_path):
     image.save(file_path)
 
 
+def get_code(driver, file_path):
+    if(not(os.path.isdir('source'))):
+        utils.mkdir('source')
+    source = driver.page_source
+    with open(file_path, "w", encoding="utf-8") as open_file:
+        open_file.append("source code : " + source + "\n")
```

Comment:
Don't append, just write.
Also don't write the "source code :" and the newline at the end, they are unnecessary.

Nit: rename open_file to f
Nit: no need to store the **driver.page_source** in a variable if you are using it only once